test/plugin/scenarios/resttemplate-4.x-scenario/support-version.list

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+4.3.8.RELEASE
```

Comment:
Does this plugin support 4.0, 4.1, 4.2? If so, please add some(not need to add all) to the tests.

Reply:
no support

Reply:
The Supported-list.md in the doc says otherwise. Is there an incompatible change? Could you explain a little about this? Is the old implementation for old versions?

Reply:
Later, I will take the time to write a non-springboot project and test it

Reply:
OK. Work for me.